cli/cmd/bb/bb.go

```diff
@@ -52,8 +52,7 @@ func run() (exitCode int, err error) {
 	// Record original arguments so we can show them in the UI.
 	originalArgs := append([]string{}, os.Args...)
 
-	// Handle global args that don't apply to any specific subcommand
-	// (--verbose, etc.)
+	// Configure verbose flag
```

Comment:
The previous comment was helpful IMO because if we want to add more CLI-global flags, this is where we should do it - otherwise the logic on line 76 (in this PR) won't necessarily work

Reply:
Added a separate helper function to parse global CLI flags - will make it more clear what flags are supported and easier to add them in the future